plugin.xml

```diff
@@ -138,6 +144,8 @@
             <uses-permission android:name="android.permission.ACCESS_NOTIFICATION_POLICY" />
             <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
             <uses-permission android:name="android.permission.WAKE_LOCK" />
+            <uses-permission android:name="android.permission.POST_NOTIFICATIONS"/>
+            <uses-permission android:name="android.permission.SCHEDULE_EXACT_ALARM" />
```

Comment:
Schedule exact alarm is no longer allowed on android 13 for non alarm apps. https://developer.android.com/about/versions/14/changes/schedule-exact-alarms#calendar-alarm-clock)

Reply:
I think we have do it in a separate PR.
@fquirin What do you think?